scripts/update-locales.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/sh
 
 mv ./options/locale/locale_en-US.ini ./options/
-sed -i -e 's/=\"/=/g' -e 's/\"$$//g' ./options/locale/*.ini
-sed -i -e 's/\\\\\\\\\"/\"/g' ./options/locale/*.ini
+
+# Make sure to only change lines that have the translation enclosed between quotes
+sed -i -r -e '/^[a-zA-Z0-9_-.]+[ ]*=[ ]*".*"$/ {
+	s/^([a-zA-Z0-9_-.]+)[ ]*="/\1=/
+	s/\\"/"/g
+	s/"$//
+	}' ./options/locale/*.ini
```

Comment:
Doesn't it count when translated string inside quote also have nested quotes?

Reply:
Nested quotes are escaped with `\`. The format to fix from is:
```
module.resource_name="<a href=\"https://gitea.com\">"
```
And we want it to be:
```
module.resource_name=<a href="https://gitea.com">
```
But most of the lines are already in the target format, so we have to be careful to not modify them.